cmd/drone-bazelisk-ecr/plugin.go

```diff
@@ -43,12 +48,32 @@ func newPlugin() (plugin, error) {
 		os.Setenv("AWS_SECRET_ACCESS_KEY", p.SecretKey)
 	}
 
-	return p, nil
+	return nil
+}
+
+func (p *plugin) getArgs() []string {
+	var args []string
+
+	// append startup options
+	if p.Bazelrc != "" {
```

Comment:
Should we test if this file exists?  Or does bazel fail if this argument points to a file that does not exist?  
If the behavior of bazel is to ignore the argument if it does not exist, we should throw an error.

Reply:
good point, i'll look into this.

Reply:
I'll ok with this being added on iteratively.

Reply:
bazelisk errors out, so leaving as is for now.

```
[ERROR 17:56:49.418 src/main/cpp/option_processor.cc:375] Error: Unable to read .bazelrc file '/drone/src/.bazelrc.drone'.
```